core/services/workflows/metering/metering.go

```diff
@@ -196,9 +195,29 @@ func (r *Report) Deduct(ref string, amount int64) error {
 	return nil
 }
 
+func (r *Report) ApplyLimitToRequest(
+	info capabilities.CapabilityInfo,
+	req *capabilities.CapabilityRequest,
+	amount int64,
+) {
+	if len(info.SpendTypes) > 0 {
+		spendType := info.SpendTypes[0]
+
+		// use rate card to convert capSpendLimit to native units
+		rate, ok := r.rateCard[string(spendType)]
```

Comment:
Can we run all rate card things through the balance store to keep the logic in one place?

My last PR to exposes the BalanceStore's conversions: `ConvertFromBalance` and `ConvertToBalance` for the Engine to use.

Reply:
Ah ok. I can use those instead.